### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by KateBlossom**
- [DEMO LINK](https://KateBlossom.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://KateBlossom.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,8 +11,48 @@
...
+        src="./images/imac.jpeg"
+        alt="Imac"
+      />
+      <div class="product-details">
+        <h2 class="product-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-code">Product code: 195434</p>
+        <div class="flex-container">
+          <div class="stars">
```
Comment (volodymyr-soltys97) [inline src/index.html:34]: Add the modifier classes
```suggestion
          <div class="stars stars--4">
```

File: src/index.html [hunk 2]
```diff
@@ -11,8 +11,48 @@
...
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <p class="product-review">Reviews: 5</p>
+          </div>
+        </div>
+        <div class="flex-container">
+          <p class="product-price">Price:</p>
+          <p class="price-value">$2,199</p>
+        </div>
+        <div class="add-to-cart">
```
Comment (volodymyr-soltys97) [inline src/index.html:47]: This div is redundant
```suggestion
```

File: src/styles/index.scss [hunk 3]
```diff
@@ -1,3 +1,136 @@
...
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
 }
+
+.product-card {
+  width: 200px;
+  height: 408px;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:11]: No need to hardcode height for the card
```suggestion
```

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,136 @@
...
+  background-color: #fff;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:12]: Default color
```suggestion
```

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,136 @@
...
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px;
+  margin: 10px;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:19]:
```suggestion
```

File: src/styles/index.scss [hunk 6]
```diff
@@ -1,3 +1,136 @@
...
+.product-card {
+  width: 200px;
+  height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 10px;
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:18]: Check padding on design and fix it

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,136 @@
...
+  text-align: center;
+  text-transform: uppercase;
+  justify-content: center;
+}
+
+.buy-link:hover {
+  border: 1px solid #00acdc;
+  color: #00acdc;
+  background-color: #f3f3f3;
+}
```
Comment (volodymyr-soltys97) [inline src/styles/index.scss:136]: Use scss nesting

**Review by volodymyr-soltys97 — CHANGES_REQUESTED** [review]
Good job 👍 
Let's improve your code
You need to remove it 
image

**Comment by KateBlossom** [comment]
Hi, I don't know why, but my previous file was uploaded, which I deleted and already created new. I will try to add a new version.

**Pull request closed without merging**